app/femr/data/models/mysql/Patient.java

```diff
@@ -38,6 +38,8 @@
     private String firstName;
     @Column(name = "last_name", nullable = false)
     private String lastName;
+    @Column(name = "phone_number", nullable = false)
```

Comment:
Why are we trying to enforce `nullable = false`? I noticed the empty string gets mapped in the datamodelmapper instead of NULL being used